.github/workflows/update-self-version-reference.yml

```
@@ -0,0 +1,64 @@
+          GITHUB_TOKEN: ${{ secrets.ADGEM_SDK_ACTIONS_TOKEN }}
+          CHECK_TYPE: 'self'
+        run: |
+          LATEST_VERSION=$(node ./bin/check-for-updates.js)
+          if [ $? -ne 0 ]; then
+            echo "Failed to get latest SDK version"
+            exit 1
+          fi
+          echo "LATEST_VERSION=$LATEST_VERSION" >> "$GITHUB_ENV"
+
+      - name: Update README.md
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:47: Double quote to prevent globbing and word splitting

(shellcheck)